### ISSUE TITLE
Backup error in 4.0.0

### ISSUE THREAD

**Issue by vkorobov** [sameersbn/docker-redmine#378]
I try new version and run backup (scheduled and manual).
```
root@9beb8f0b7114:/home/redmine/redmine# entrypoint.sh app:backup:create
Initializing logdir...
Initializing datadir...
Symlinking dotfiles...
Installing configuration templates...
Configuring redmine...
Configuring redmine::database
Configuring redmine::unicorn...
Configuring redmine::secret_token...
Configuring redmine::max_concurrent_ajax_uploads...
Configuring redmine::sudo_mode...
Configuring redmine::autologin_cookie...
Configuring redmine::email_delivery...
Configuring redmine::incoming_email...
Configuring redmine::backups...
Configuring nginx...
Configuring nginx::redmine...
Dumping PostgreSQL database redmine_production...
pg_dump: [archiver (db)] connection to database "redmine_production" failed: FATAL:  password authentication failed for user "redmine"
```

The same with MySQL database
```
Dumping MySQL database redmine_production...
mysqldump: Got error: 1045: Access denied for user 'redmine'@'172.22.0.3' (using password: YES) when trying to connect
```
On previous version 3.4 backup is work.

**Comment by jcormier** [comment]
``` 
root@cde9006906fa:/home/redmine/redmine# bash -x entrypoint.sh app:backup:create
...
+ eval 'database_production_adapter="postgresql"' 'database_production_encoding="unicode"' 'database_production_database="redmine_production"' 'database_production_host="postgresql"' 'database_production_port="5432"' 'database_production_username="redmine"' 'database_production_password="'\''password'\''"' 'database_production_pool="5"'
++ database_production_adapter=postgresql
++ database_production_encoding=unicode
++ database_production_database=redmine_production
++ database_production_host=postgresql
++ database_production_port=5432
++ database_production_username=redmine
++ database_production_password=''\''password'\'''
++ database_production_pool=5
+ case ${database_production_adapter} in
+ echo 'Dumping PostgreSQL database redmine_production...'
Dumping PostgreSQL database redmine_production...
+ PGPASSWORD=''\''password'\'''
+ pg_dump --clean --host postgresql --port 5432 --username redmine redmine_production
pg_dump: [archiver (db)] connection to database "redmine_production" failed: FATAL:  password authentication failed for user "redmine"

 root@cde9006906fa:/home/redmine/redmine# pg_dump --clean --host postgresql --port 5432 --username redmine redmine_production > /dev/null; echo $?
Password: 
0
root@cde9006906fa:/home/redmine/redmine# 
root@cde9006906fa:/home/redmine/redmine# PGPASSWORD=password pg_dump --clean --host postgresql --port 5432 --username redmine redmine_production > /dev/null; echo $?
0

root@cde9006906fa:/home/redmine/redmine# PGPASSWORD="\"password"\" pg_dump --clean --host postgresql --port 5432 --username redmine redmine_production > /dev/null; echo $?
+ PGPASSWORD='"password"'
+ pg_dump --clean --host postgresql --port 5432 --username redmine redmine_production
pg_dump: [archiver (db)] connection to database "redmine_production" failed: FATAL:  password authentication failed for user "redmine"
+ echo 1
1
root@cde9006906fa:/home/redmine/redmine# PGPASSWORD="password" pg_dump --clean --host postgresql --port 5432 --username redmine redmine_production > /dev/null; echo $?
+ PGPASSWORD=password
+ pg_dump --clean --host postgresql --port 5432 --username redmine redmine_production
+ echo 0
0
```
Looks like how the password is quoted is affecting the pg_dump login.  Looking into it.

This is likely from the change for this issue #373

**Comment by jcormier** [comment]
Pushed an update to master.  Can you test it? Worked for me.

```
 $ docker-compose run redmine app:backup:create
Starting dockerredmine_postgresql_1 ... 
Starting dockerredmine_postgresql_1 ... done
Initializing logdir...
Initializing datadir...
Symlinking dotfiles...
Installing configuration templates...
Configuring redmine...
Configuring redmine::database...
Configuring redmine::unicorn...
Configuring redmine::secret_token...
Generating a session token...
Note:
  All old sessions will become invalid.
  Please specify the REDMINE_SECRET_TOKEN parameter for persistence.
  **SHOULD** be defined if you have a load-balancing Redmine cluster.
Configuring redmine::max_concurrent_ajax_uploads...
Configuring redmine::sudo_mode...
Configuring redmine::autologin_cookie...
Configuring redmine::backups...
Configuring nginx...
Configuring nginx::redmine...
Dumping PostgreSQL database redmine_production...
Dumping files...
Dumping dotfiles...
Dumping plugins...
Dumping themes...
Creating backup archive: 1547831926_redmine_backup.tar...
```

**Comment by vkorobov** [comment]
Work fine now!
Thanks

**Comment by jcormier** [comment]
Released version 4.0.0
